Cards/CardBase.gd

```diff
@@ -50,25 +42,9 @@ func can_play_card(caster: Entity, target: Entity) -> bool:
 
 func on_card_play(caster: Entity, target: Entity) -> void:
 	_apply_all_effects(target)
-	_deal_damage(caster, target)
-	_apply_status(caster, target)
-	_draw_cards()
-	_discard_random_cards()
 	CardManager.on_card_action_finished.emit(self)
 	# TODO add other functionality that lots of cards may share (eg: restore AP)
 
-
-# override in child cards if you want to deal damage in a unique way
-func _deal_damage(caster: Entity, target: Entity) -> void:
-	# damage target
-	if damage_to_apply_to_target != 0.0:
-		_damage_entity(caster, target, damage_to_apply_to_target, affect_all_targets)
-	
-	#damage caster
-	if damage_to_apply_to_caster != 0.0:
-		_damage_entity(caster, caster, damage_to_apply_to_caster, affect_all_casters)
-
-
 func _damage_entity(caster: Entity, target: Entity, damage_amount: float, damage_all: bool) -> void:
```

Comment:
looking at my own PR, this got deleted. i think I put the effect in the `EffectDamage` card. Would like confirmation from someone that reviewed my initial PR (@Tysterman74 or @Tomzkk ) but i'm pretty sure you can also delete this function from CardBase

Reply:
I can see if anything breaks if I remove it

Reply:
Ye, it seems like it's fine without so I'll push that

Reply:
perfect thanks !